frontend/Themes/Themes.js

```diff
@@ -450,6 +474,7 @@ module.exports = {
   Eighties,
   FirefoxDark,
   FirefoxLight,
+  Firebug,
```

Comment:
nit: Let's name this `FirefoxFirebug` to keep it better grouped with the other Firefox themes?